chromium_src/third_party/blink/renderer/core/frame/local_frame.cc

```diff
@@ -107,4 +114,13 @@ SkBitmap LocalFrame::GetImageAtViewportPoint(const gfx::Point& viewport_point) {
   return {};
 }
 
+bool LocalFrame::ScriptEnabled(const KURL& script_url) {
```

Comment:
I don't believe this functionality is even relevant anymore cc @diracdeltas

Reply:
discussed on slack. we do still allow users to enable specific scripts when they block scripts on a page. but it's hard to find so i'm not against just removing this functionality.

Reply:
sounds like we are going to check usage with p3a before disabling so we can leave this for now